src/xinterpreter.cpp

```diff
@@ -348,9 +348,12 @@ namespace xcpp
         va_start(args, format);
 
         int ret;
-        if (stream == stderr) {
+        if (stream == stdout || stream == stderr) {
             std::string buf = c_format(format, args);
-            std::cerr << buf;
+            if (stream == stdout)
+                std::cout << buf;
+             else if (stream == stderr)
+                std::cerr << buf;
```

Comment:
Sorry for the nitpicking comment!

Would you mind adding curly brackets for if / else statements even if it is just for one line?

Reply:
Thx :) We are a bit obsessive about this :)

Reply:
Done. I'm getting confused about styles in different projects I contribute to...